src/Grpc.NetCore.HttpClient/Grpc.NetCore.HttpClient.csproj

```diff
@@ -12,15 +12,17 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <LangVersion>8.0</LangVersion>
     <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- Roslyn analyzer that enforces ConfigureAwait(false) -->
+    <PackageReference Include="ConfigureAwaitChecker.Analyzer" Version="3.0.0" PrivateAssets="All" />
```

Comment:
Use dependency.props to manage the version for this dependency.

Reply:
Also, are we allowed to use this tool? For our repos, we have strict requirements on which third party tools we were allowed to use.

Reply:
`PrivateAssets="All"` = build time dependency. ASP.NET Core will never see it.

@jtattermusch Do you have any issues with using this tool? It's MIT, and a Roslyn analyzer that can be removed at anytime. I want to use it, or something like it, to ensure we have `ConfigureAwait(false)` everywhere. See https://stackoverflow.com/questions/27851073/why-would-i-bother-to-use-task-configureawaitcontinueoncapturedcontext-false/27851460#27851460

Reply:
I'm fine with using that analyzer.